content/en/monitors/monitor_types/composite.md

```diff
@@ -75,11 +76,12 @@ a || b && !c
 
 For detailed instructions on the **Say what's happening** and **Notify your team** sections, see the [Notifications][2] page.
 
-## API
+### API
 
-When using the API, a non-composite monitor's query can encapsulate a metric, tags, aggregation function like `avg`, group-by clause, etc. A composite monitor's query is defined in terms of its constituent monitors using monitor IDs.
+When using the API, a composite monitor's query is defined in terms of its constituent monitors using monitor IDs.
```

Comment:
```suggestion
When using the API, a composite monitor's query is defined in terms of its constituent monitors using monitor IDs.
```
I am not sure what you mean here by constituent monitors? Dependent monitors? Related monitors?

Reply:
Used the same word as before but we definitely intent to say "Child monitor" or sub-monitor here.